elastic/elastic-agent
# Cloudnative ci utomation

## What does this PR do?

Continuation of https://github.com/elastic/elastic-agent/pull/837

## Why is it important?

Fixes the SLA check
https://github.com/elastic/kibana/pull/138713#issuecomment-1212903002

## Checklist




## Author's Checklist



## How to test this PR locally



- See automatic PR with fixed SLA https://github.com/elastic/kibana/pull/139630#issuecomment-1230262241
- Successful Jenkins job:https://fleet-ci.elastic.co/job/elastic-agent/job/elastic-agent-mbp/view/change-requests/job/PR-1035/2/console

## Related issues


- Relates https://github.com/elastic/elastic-agent/issues/706

## Use cases



## Screenshots



## Logs
